crates/locked-app/src/lib.rs

```diff
@@ -0,0 +1,50 @@
+//! Spin internal application interfaces
+//!
+//! This crate contains interfaces to Spin application configuration to be used
+//! by crates that implement Spin execution environments: trigger executors and
+//! host components, in particular.
+
+#![deny(missing_docs)]
+
+pub mod locked;
```

Comment:
{C,Sh}ould this module be flattened into the crate root? (perhaps be `pub use`ing all its interesting bits)

Reply:
It could; again, this seemed like a way to minimise code changes elsewhere.  Most Spin crates continue to use the full-fat `spin-app` crate, and it just seemed easier to keep things very much the same.